tests/test_ops/test_onnx.py

```diff
@@ -182,3 +183,26 @@ def warpped_function(torch_input, torch_rois):
         # allclose
         os.remove(onnx_file)
         assert np.allclose(pytorch_output, onnx_output, atol=1e-3)
+
+
+def test_simplify():
+    try:
+        from mmcv.onnx import simplify
+    except ImportError:
```

Comment:
What's this case?

Reply:
Would be removed. Thought for backward compatibility.